tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ES2022",
     "module": "es2022",
-    "moduleResolution": "node",
```

Comment:
The import from `vscode-test-playwright` won't work unless this change is made, for whatever reason. I hope it's not very disruptive.

Reply:
Am having trouble finding what the _default_ setting is, but [it does read](https://www.typescriptlang.org/tsconfig/#moduleResolution) that 'node' mode is antiquated and 'node16' is what 'new' projects against node should be using. Am going to try to cook this branch with it explicitly set to 'node16' to see if happy.